xwiki-platform-core/xwiki-platform-user/xwiki-platform-user-profile/xwiki-platform-user-profile-ui/src/main/resources/XWiki/XWikiUserSheet.xml

```diff
@@ -578,6 +578,40 @@ div.editProfileCategory {
   float: right;
 }
 
+/* Preferences */
+/* Shortcut preferences */
+span.xHint:has(+ ul.shortcut-list) {
+  margin-bottom: 10px;
+}
+
+ul.shortcut-list {
+  list-style: none;
+  padding-left: unset;
+  &amp; &gt; li {
+    display: flex;
+    flex-direction: row;
+    gap: 1em;
+    justify-content: space-between;
+    align-items: center;
+    /* Keep the line large enough to fit an input in both view and edit mode. */
+    height: 38px;
```

Comment:
Doesn't this clash with the idea that font size is variable?

Reply:
Since this is a CSS file, I decided to hardcode this instead of relying on LESS variables.
Using LESS variables, it would have been quite a lot of operations to get this result:
max(@line-height-base*@font-size-base+ 2*@padding-base-vertical + [2*2](https://github.com/xwiki/xwiki-platform/blob/0fbc4ccbae83e56f8fbff12964f53a97b6c0ecda/xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/less/buttons.less#L92-L93) + 2*1 px, @input-height-base + 2*1 px)
We want to fit both the button and the input in Edit mode, that both use different variables for their height...

The only impact of defining this height is the min-height. With different sizes of font sizes:
* Larger font sizes: the edit mode lines are larger than the view mode ones, in order to fit the large inputs.
* Smaller font sizes: the input and buttons have more space around them in edit mode.

I [changed the style](https://github.com/xwiki/xwiki-platform/pull/2981/commits/72e7f894a2e4f081c40d4ee916f351a0e591bef3) to make it clearer that this is not an upper limit on the height of the lines. If the content is large, the line height will increase to fit this content :) 

Those are very small drawbacks to hardcoding. I'd rather keep it hardcoded with a comment rather than spend time trying to get something to work with our current LESS and _need_ to rework it soon.

Reply:
Can't you use a height based on `em`?